### PR TITLE
Criteria admin panel

### LINKED ISSUE #24: Criteria admin panel
Reference to data structure issue for clarification.

_Originally posted by @amundlrohne in https://github.com/navikt/innbyggerpanelet/issues/20#issuecomment-1011928365_

### PR DESCRIPTION
**Pull request by amundlrohne**
Closes #24

### CODE REVIEW

**Comment by amundlrohne** [comment]
Is dependent on #16

File: apps/frontend/src/app/containers/criteriaAdministration/CriteriaAdminPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                    </Accordion>
+                ))}
+            </Panel>
+            <CriteriaEditModal
+                criteria={editCriteria}
+                open={editCriteria != undefined}
+                close={() => setEditCriteria(undefined)}
+            />
+        </>
+    );
```
Comment (perkynades) [inline apps/frontend/src/app/containers/criteriaAdministration/CriteriaAdminPanel.tsx]: Here there is a lot happening, split it up into seperate components

File: libs/api-interfaces/src/lib/api-interfaces.ts [hunk 2]
```diff
@@ -20,6 +25,14 @@ export interface Candidate {
 export interface Trait {
     id: number;
     name: string;
+    exclusivitySlug?: string;
+}
+
+export interface CriteriaCategory {
+    id: number;
+    name: string;
+    description: string;
```
Comment (perkynades) [inline libs/api-interfaces/src/lib/api-interfaces.ts:46]: Remember to update these :D

**Review by perkynades — CHANGES_REQUESTED** [review]
Looks very good, there are just some small things u need to change before merging :)

**Pull request closed without merging**